install/kots/manifests/kots-app.yaml

```diff
@@ -33,14 +33,14 @@ spec:
     # @link https://www.gitpod.io/docs/self-hosted/latest/monitoring
     - title: Running Workspaces
       queries:
-        - query: 'sum(gitpod_ws_manager_workspace_phase_total{phase="RUNNING"}) by (type)'
+        - query: 'sum(gitpod_ws_manager_mk2_workspace_phase_total{phase="Running"}) by (type)'
```

Comment:
dont think this is used anymore, but just updated every reference I could find in the repo